lib/WP_Auth0_ErrorManager.php

```diff
@@ -19,9 +19,12 @@ public static function insert_auth0_error( $section, $error ) {
 		} elseif ( $error instanceof Exception ) {
 			$code = $error->getCode();
 			$message = $error->getMessage();
+		} elseif ( is_array( $error ) && ! empty( $error['response'] ) ) {
+			$code = ! empty( $error['response']['code'] ) ? $error['response']['code'] : 'N/A';
```

Comment:
Instead of `N/A` use something like `a0_unknown_code` or if you received a "plain text response" then use that  as description and use some code like `a0_plaintext_error`